src/l500/l500-options.cpp

```diff
@@ -36,6 +36,17 @@ namespace librealsense
         return _range;
     }
 
+    float l500_hw_options::get_default() const
+    {
+        auto current = query( int( _resolution->query() ) );
+        _hw_monitor->send( command{ AMCSET, _type, -1 } );
+        auto def = query( int( _resolution->query() ) );
```

Comment:
You're going to add the part about the lag if streaming?

Reply:
yes